### ISSUE TITLE
Add screenshots of the project in the readme.md .

### ISSUE THREAD

**Issue by gaharavara** [gaharavara/WEB-DEP#1]
Add the screenshots of the project under a proper heading included in the readme.md

Important note: Please make sure the screen shot includes any basic html, css, js code example showing the website working.

Please comment your doubts if any. ( Basically for the first time contributors )

**Comment by Mr-emeka** [comment]
Hello @gaharavara I would like to be assigned to this task.

**Comment by gaharavara** [comment]
Go for it.

**Comment by AmirMehrabi** [comment]
I also would like to contribute to this project. Not only on creating the documentation, but also on making it better in general.

**Comment by gaharavara** [comment]
@AmirMehrabi  All ideas for improving the project are welcome, please do create a seperate issue for any other change or feature implementation you want to do. 

Also do comment the portions you add and include a proper description for the changes you intend to do.

**Comment by Geekz45679** [comment]
I've made a [PR](https://github.com/gaharavara/WEB-DEP/pull/3), let me know what you think!

**Comment by AmirMehrabi** [comment]
> @AmirMehrabi All ideas for improving the project are welcome, please do create a seperate issue for any other change or feature implementation you want to do.
> 
> Also do comment the portions you add and include a proper description for the changes you intend to do.

Will do my man.

**Comment by Mr-emeka** [comment]
@gaharavara the whole thing doesn't seem to work

**Comment by gaharavara** [comment]
@Mr-emeka yes you are running the right file, I am not sure what error you are facing. A detailed description of the error would be much appreciated 👍
